hazelcast/src/test/java/com/hazelcast/jet/impl/submitjob/clientside/upload/JobUploadClientFailureTest.java

```diff
@@ -61,12 +62,17 @@
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.doAnswer;
 
+// Tests to upload jar to member
 @RunWith(HazelcastSerialClassRunner.class)
 @Category({QuickTest.class})
 public class JobUploadClientFailureTest extends JetTestSupport {
 
+    // The newJob() is called from main thread
```

Comment:
Add in-line comments -> JavaDocs

Reply:
rev :35bc7755ad47b74ac4e7f21e3a3e9a0fa9f37144